src/nupic/algorithms/TemporalMemory.cpp

```diff
@@ -1025,6 +1025,20 @@ bool TemporalMemory::operator==(const TemporalMemory &other) const {
 // Debugging helpers
 //----------------------------------------------------------------------
 
+
```

Comment:
PR #490 also added an operator>> so there might be a merge conflict.

Reply:
Thanks for reviewing this!

I just reviewed your `operator<<` implementations, and yes there will be a conflict.  I think yours have more information, but also that they need a bit more work to make a concise and useful diagnostic print out.